### PR TITLE
[TW-875] Doc Improvements to Public Documentation

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/publishing-your-api/publishing-your-docs.md [hunk 1]
```diff
@@ -63,9 +62,22 @@ To publish the documentation for a collection, do the following:
...
     * **Custom domain** - Select a [custom domain](/docs/publishing-your-api/custom-doc-domains/) where you want to publish your documentation.
     * **Default layout** - Select a layout style for your documentation. **Double column** displays sample code in a column next to the documentation. **Single column** displays sample code inline beneath each request.
-    * **Header background color** - Specify a color (hex format) for the header at the top of the documentation window.
-    * **Code background color** - Specify a color (hex format) for sample code blocks.
-    * **Highlight color** - Specify a color (hex format) for hyperlinks.
+    * **Default Theme** - Select a light or dark theme for your documentation. You can also choose to use the system theme. Your documentation will use the selected theme by default, but users can switch between themes.
+    * **Logo** - By default, public documentation uses your [team logo](/docs/administration/team-settings/#editing-your-team-profile). You can also select a custom logo for each theme (light and dark):
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/publishing-your-api/publishing-your-docs.md:66]: Great suggestions! I've updated the note as follows:

> You can use different logos for each collection you publish. Logos must be 2 MB or less in size and must be in jpg, jpeg, or png format. The logo can be any aspect ratio (square or rectangle.)

**Review by Himanshu0781 — APPROVED** [review]

**Comment by Himanshu0781** [comment]
Looks good ✅

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/publishing-your-api/publishing-your-docs.md [hunk 2]
```diff
@@ -63,9 +62,24 @@ To publish the documentation for a collection, do the following:
...
+
+        * To add a logo, select the edit icon <img alt="Edit icon" src="https://assets.postman.com/postman-docs/documentation-edit-icon-v8-10.jpg#icon" width="18px"> and select **Upload**. Drag and drop an image file, or select an image file. Drag the handles to adjust the part of the image you want to display, then select **Upload**.
+        * To delete a logo, select the edit icon <img alt="Edit icon" src="https://assets.postman.com/postman-docs/documentation-edit-icon-v8-10.jpg#icon" width="18px"> and select **Delete**.
+
+        > You can use different logos for each collection you publish. Logos must be 2 MB or less in size and must be in jpg, jpeg, or png format. The logo can be any aspect ratio (square or rectangle.)
```
Comment (akinard-postman) [inline src/pages/docs/publishing-your-api/publishing-your-docs.md:71]: Should the formats use .-format, like .jpg or .png (like in https://learning.postman.com/docs/administration/team-settings/)?

**Review by akinard-postman — COMMENTED** [review]
Just the one question, but otherwise LGTM!

**Review by akinard-postman — APPROVED** [review]

File: src/pages/docs/publishing-your-api/publishing-your-docs.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/publishing-your-api/publishing-your-docs.md]: Thanks! I had misunderstood the direction in our style guide. Turns out we're supposed to follow the Google style guide on this, which means it should be JPEG and PNG
https://developers.google.com/style/filenames?hl=en#file-type-names

Comment (akinard-postman) [reply src/pages/docs/publishing-your-api/publishing-your-docs.md]: Thanks for the head's up @mark-anthony-postman !  I was a bit unclear myself so I fell back on what I saw elsewhere in the Learning Center. Good to know. :D

Comment (mark-anthony-postman) [reply src/pages/docs/publishing-your-api/publishing-your-docs.md]: Yes, we'll probably need to update these elsewhere in the LC at some point 😀

**Pull request merged**